.vsts-ci.yml

```diff
@@ -0,0 +1,149 @@
+resources:
+- repo: self
```

Comment:
How was this file generated?

Reply:
This is generated initially by using the "View YAML" link on our old VSTS build definition. It's been altered a bit from that initial definition though.